app/views/partials/code_and_builds.scala.html

```diff
@@ -33,7 +33,7 @@ <h3 class="board__heading">Code and Build</h3>
                 <li class="list-item"><a id="link-to-@{ciLink.id}" href="@{ciLink.url}" target="_blank">@{ciLink.displayName}<span class="glyphicon glyphicon-new-window"/></a></li>
             }
             @if(repo.repoType == RepoType.Service) {
-                <li class="list-item"><a id="link-to-dependency-explorer" href="@{DependenciesController.service(repo.name)}" target="_blank">Service Dependencies<span class="glyphicon glyphicon-new-window"/></a></li>
+                <li class="list-item"><a id="link-to-dependency-explorer" href="@{DependenciesController.services(repo.name)}" target="_blank">Service Dependencies<span class="glyphicon glyphicon-new-window"/></a></li>
```

Comment:
rename `link-to-dependency-explorer` since it's a different dependency explorer?